owncloudApp/src/main/res/values/strings.xml

```diff
@@ -590,6 +590,8 @@
     <string name="share_via_link_expiration_date_label">Expiration</string>
     <string name="share_via_link_password_label">Password</string>
     <string name="share_via_link_password_title">Secured</string>
+    <string name="share_via_link_generate_password_button">Generate password</string>
+    <string name="share_via_link_copy_password_button">Copy</string>
```

Comment:
This could be sorted alphabetically. Maybe in another issue...

Reply:
No sense to order strings alphabetically, they should be grouped by meaning. In any case, `strings.xml` is a chaos right now, this PR is obviously not the scope to work on it